students/PeterB/Session3/list_lab.py

```diff
@@ -0,0 +1,63 @@
+#series 1
+
+Fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print('Fruits')
+
+New_Fruit = input('Banana')
+Fruits.append(New_Fruit)
+print(Fruits)
+
+Number = input('Pick a number')
+print(f'the {Fruits} in the list is [int(Fruits)-1]')
```

Comment:
This doesn't print out what were expecting here. You want to print out the i-th item given in the list, when given i+1. So for example, if the user returns 4, you want to print fruits[3].